### PR TITLE
completed prep-example Nicholas Cardoza

### PR DESCRIPTION
**Pull request by NicholasCardoza**
Screenshot 2022-11-12 at 10 30 35 AM

### CODE REVIEW

File: prep-example/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<head>
+  <title>prep-example</title>
+</head>
+<body>
+  <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+
+
+
```
Comment (thebearingedge) [inline prep-example/index.html]: Hey Nick! When you have a sec, it looks like your text was not included for the commit

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-example/index.html (hunk 1 above)
Comment (NicholasCardoza) [reply prep-example/index.html]: Of corse I deleted the wrong one

Comment (NicholasCardoza) [reply prep-example/index.html]: I imported 2 of the same example files into vsc

**Comment by NicholasCardoza** [comment]
Screenshot 2022-11-12 at 1 28 06 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**